src/main/java/emu/grasscutter/game/managers/energy/EnergyManager.java

```diff
@@ -152,12 +187,19 @@ public void handleGenerateElemBall(AbilityInvokeEntry invoke) throws InvalidProt
         if (action == null) {
             return;
         }
-
         // Default to an elementless particle.
-        int itemId = 2024;
+        int itemId = elementTypeDefault;
 
         // Generate 2 particles by default.
-        int amount = 2;
+        int amount = particlesDefault;
+        if(invoke.getHead().getLocalId()==98307){
+            amount = 3;
+            for (int i = 0; i < amount; i++) {
+                generateElemBall(itemId, new Position(action.getPos()), 1);
+            }
+            return;
+        }
```

Comment:
What is that? A fix for Favonius? Please put a discriptive comment if you handle a specific special case.